packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -129,31 +129,16 @@ This app may not exist or you may not have permission to view it.''',
     }
   }
 
-  /// Exits if [platform] release artifacts already exist for an
-  /// [existingRelease].
-  Future<void> ensureReleaseHasNoArtifacts({
-    required String appId,
-    required Release existingRelease,
+  /// Checks whether [release] is in an active state for [platform]. If so,
+  /// exits with code 70.
+  void ensureReleaseIsIsNotActive({
```

Comment:
```suggestion
  void ensureReleaseIsDraft({
```

Reply:
This isn't quite true – a release may not have a status for the given platform

Reply:
Ah I see in that case can we remove the extra `Is` -> `ensureReleaseIsNotActive`

Reply:
Hahaha yes we can 😳